app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt

```diff
@@ -52,50 +49,66 @@ class PinPasswordActivityPresenter @Inject constructor(
       viewModel = pinViewModel
     }
 
+    binding.pinPassordToolbar.setNavigationOnClickListener {
+      (activity as PinPasswordActivity).finish()
+    }
```

Comment:
Why is this needed now? It doesn't seem like we did it this way before.

Reply:
In older UI we were not having toolbar. In new version of Pin VIew UI toolbar is been introduced.